### DISCUSSION TITLE
change random number providers

### DISCUSSION THREAD

**Discussion by CANEFIL** [betfinio/public#111] [Bugs and Reports]
we still have the random number provider Chainlink on the site for roulette, but we have changed it to Gelato. We should change it now that we've communicated this information to the people. 

![Screen Shot 2024-12-27 at 12 39 46 PM](https://github.com/user-attachments/assets/71ebd10a-fdc5-409c-bbc0-c49e959a13c9)

**Comment by bf-monster** [comment]
Thank you for the update, we will add it to backlog!
